app/src/main/java/net/gsantner/opoc/frontend/filebrowser/GsFileBrowserListAdapter.java

```diff
@@ -468,8 +486,6 @@ public void loadFolder(final File folder) {
             @Override
             public void run() {
                 synchronized (LOAD_FOLDER_SYNC_OBJECT) {
-                    ArrayList<File> oldAdapterData = new ArrayList<>();
-                    Collections.copy(_adapterData, oldAdapterData);
```

Comment:
These lines did nothing?

Reply:
They did and it was keeping for data backup. But I generified the logic in GsFileUtils so this is leftover ^^